Content.Client/Input/ContentContexts.cs

```diff
@@ -86,7 +86,7 @@ public static void SetupContexts(IInputContextContainer contexts)
             human.AddFunction(ContentKeyFunctions.Arcade2);
             human.AddFunction(ContentKeyFunctions.Arcade3);
 
-            // actions should be common (for ghosts, mobs, etc)
+            // actions should be common (for ghosts, mobs, etc.)
```

Comment:
Should be etc. but this is an upstream line of code so best left unchanged for merge conflicts
```suggestion
            // actions should be common (for ghosts, mobs, etc)
```